openmindat/mindat_api.py

```diff
@@ -64,19 +67,19 @@ def __init__(self):
         self.data_dir = './mindat_data/'
         Path(self.data_dir).mkdir(parents=True, exist_ok=True)
 
-    def load_api_key(self):
+    def load_api_key(self): 
```

Comment:
for the load_api_key function, please follow this procedure:
> import os
if os.environ.get("MINDAT_API_KEY"):
    return os.environ.get("MINDAT_API_KEY")
else:
try:
with open('./.apikey.yaml', 'r') as f:
                data = yaml.safe_load(f)
            return data['api_key']
        except FileNotFoundError:
            return ''